thoth/storages/graph/postgres.py

```diff
@@ -4761,9 +4761,13 @@ def _create_python_package_version(
             os_version=os_version,
             python_version=python_version,
             entity_id=entity.id,
-            python_package_metadata_id=python_package_metadata_id,
         )
 
+        # including this value in "get_or_create" will cause errors because it is not part of a unique entry
+        if python_package_metadata_id is not None:
+            python_package_version.python_package_metadata_id = package_metadata.id
```

Comment:
`package_metadata` is not passed to this method

Reply:
ty